buildSrc/src/main/java/Dependency.kt

```diff
@@ -30,6 +30,8 @@ object Dependency {
     const val ACCOMPANIST_PLACEHOLDER =
         "com.google.accompanist:accompanist-placeholder:${Version.ACCOMPANIST}"
     const val ACCOMPANIST_PAGER = "com.google.accompanist:accompanist-pager:${Version.ACCOMPANIST}"
+    const val ACCOMPANIST_SWIPE_REFRESH =
+        "com.google.accompanist:accompanist-swiperefresh:${Version.ACCOMPANIST}"
```

Comment:
@suho Here is the official alternative approach using Component from `androix.compose.material`. Could you please try [this](https://developer.android.com/reference/kotlin/androidx/compose/material/pullrefresh/package-summary)?

Reply:
@hoangnguyen92dn It supports Compose from 1.3.0, while the current version of Compose on my project is 1.2.1 😁

Reply:
@suho Oh...I thought you are using the latest one. So that's fine 🙆‍♂️